src/main/java/de/tum/in/www1/artemis/domain/notification/SingleUserNotificationFactory.java

```diff
@@ -44,4 +48,40 @@ public static SingleUserNotification createNotification(Post post, NotificationT
         }
         return notification;
     }
+
+    /**
+     * Creates an instance of PlagiarismNotification
+     *
+     * @param plagiarismComparisonId ID of the plagiarism comparison associated with this case.
+     * @param recipient              affected student
+     * @param author                 instructor creating notification
+     * @param message                instructors message
+     * @return the PlagiarismNotification
+     */
+    private static SingleUserNotification createPlagiarismNotification(Long plagiarismComparisonId, Long courseId, User recipient, User author, String message, boolean isUpdate) {
+        String title = isUpdate ? "Update on your plagiarism case" : "Possible case of Plagiarism";
```

Comment:
The notification titles are string constants that are bidirectional mapped to their corresponding NotificationType(s). This is necessary for e.g. the settings/filters to work properly. Please have a look at how the other notifications (especially in the GroupNotificationFactory are managed/created) (and have a look at the `NotificationTitleTypeConstants.java` class)